### PR TITLE
fixes #6855 - bump safemode gem version

### PR DESCRIPTION
**Pull request by stbenjam**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -22,7 +22,7 @@ gem 'foreigner', '~> 1.4.2'
 
 if RUBY_VERSION =~ /^1\.8/
   # Older version of safemode for Ruby 1.8, as the latest causes regexp overflows (#2100)
-  gem 'safemode', '~> 1.0.1'
+  gem 'safemode', '~> 1.0.2'
```
Comment (dmitri-d) [inline Gemfile:25]: This is superfluous.

Comment (stbenjam) [inline Gemfile:25]: @witlessbird Thanks, understand now.  I think the latest update should do what I want?

Comment (dmitri-d) [inline Gemfile:25]: nm, it's not!

**Comment by dmitri-d** [comment]
tested under ruby 2.0.0, works.

**Comment by stbenjam** [comment]
Are you ok with the latest or should I go back to the original with the crazy version operator?

**Comment by dmitri-d** [comment]
sry to be a pain, could you revert to the original version spec?

**Comment by stbenjam** [comment]
@witlessbird Sure, done

**Comment by domcleal** [comment]
Merged as 00b1226, thanks @stbenjam!

**Pull request closed without merging**